app/services/enqueue_index_jobs.rb

```diff
@@ -0,0 +1,36 @@
+# Indexes a book version thru the SQS queuing system
+#
+# @params indexing_version e.g., Search::BookVersions::I1::IndexingStrategy::VERSION
+class EnqueueIndexJobs
+  def initialize(indexing_version:)
+    @indexing_version = indexing_version
+  end
+
+  def process_rex_releases
+    rex_releases = OpenStax::RexReleases.new
+    books_ids = rex_releases.map{ |release| release.books }.flatten
+
+    books_ids.each do |book_id|
+      process_book(book_id)
+    end
+  end
+
+  private
+
+  def process_book(book_id)
+    book_indexing = BookIndexing.create(book_version_id: book_id,
+                                        indexing_version: @indexing_version)
+
+    TodoJobsQueue.new(book_version_id: book_id,
+                                 indexing_version: @indexing_version).write
+
+  rescue => ex
+    book_indexing.destroy if book_indexing
+
+    if ex.is_a?(Dynamoid::Errors::DocumentNotValid)
+      Rails.logger.info "Book version #{book_id} already being indexed by IndexingJob"
```

Comment:
In fact it may have been indexed weeks ago, so we don't want to log this (would be a lot of duplicated log messages for what is an expected situation)